_includes/footer.html

```diff
@@ -4,7 +4,8 @@
             <ul class="list-inline">
                 <li><a href="https://github.com/namecoin"><i class="fa fa-github fa-2x" title="GitHub">GitHub</i></a></li>
                 <li><a href="https://twitter.com/Namecoin"><i class="fa fa-twitter fa-2x" title="Twitter">Twitter</i></a></li>
-                <li><a href="https://pay.reddit.com/r/namecoin"><i class="fa fa-users fa-2x" title="Reddit">Reddit</i></a></li>
+                <li><a href="https://reddit.com/r/namecoin"><i class="fa fa-reddit-alien fa-2x" title="Reddit">Reddit</i></a></li>
+                <li><a href="https://forum.namecoin.org"><i class="fa fa-comments fa-2x" title="Forum">Forum</i></a></li>
```

Comment:
Can you add a trailing forward-slash to the URL?  (My OCD dislikes URL's without a slash after the domain name....)